src/textual/widgets/_select.py

```diff
@@ -305,6 +305,46 @@ def __init__(
         self._value = value
         self._setup_variables_for_options(options)
 
+    @classmethod
+    def from_values(
+        cls,
+        options: Iterable[SelectType],
```

Comment:
Sorry to only catch this now, but if the method is called `from_values`, doesn't it make sense that the parameter is called `values`?

Also, looking at what is done, we _are_ passing in the values, so the class method name is correct and the parameter name should be updated.

Reply:
I've fixed/changed this and addressed the other comments in the 2 most recent commits.